vm/api/src/result/DumpStateResult.ts

```diff
@@ -0,0 +1,16 @@
+import type { DumpStateError } from '../errors/DumpStateError.js'
+import type { SerializableTevmState } from '@tevm/state'
+
+/**
+ * Result of Account Action
+ */
+export type DumpStateResult<ErrorType = DumpStateError> = {
+	/**
+	 * The serialized tevm state
+	 */
+	state: SerializableTevmState
```

Comment:
```suggestion
	state: SerializableTevmState
```
```suggestion
	state: SerializeToJson<SerializableTevmState>
```
The json rpc version cannot have bigints

Reply:
Ignore comment this is a result not a json rpc response